Lib/test/test_capi/test_unicode.py

```diff
@@ -9,10 +9,26 @@
     _testcapi = None
 
 
+NULL = None
+
 class CAPITest(unittest.TestCase):
 
-    # Test PyUnicode_FromFormat()
+    @support.cpython_only
+    @unittest.skipIf(_testcapi is None, 'need _testcapi module')
+    def test_fromobject(self):
+        """Test PyUnicode_FromObject()"""
+        from _testcapi import unicode_fromobject as fromobject
```

Comment:
You might check with str subclass and check that the result is not the same object.

Reply:
Done.